doc/releases/release-notes-4.0.rst

```diff
@@ -309,6 +410,10 @@ Documentation
 * Added two new build commands, ``make html-live`` and ``make html-live-fast``, that automatically
   locally host the generated documentation. They also automatically rebuild and rehost the
   documentation when changes to the input ``.rst`` files are detected on the filesystem.
+* Added two new build commands, ``make html-live`` and ``make html-live-fast``, that automatically locally
+   host the generated documentation. They also automatically rebuild and rehost the documentation when changes
+   to the input ``.rst`` files are detected on the filesystem.
+* Improved the documentation of dt_chosen_has_compat as an argument was missing
```

Comment:
definitely not something worth mentioning in release notes :) drop the entire change since you're also introducing a duplicated bullet item with "Added two new build ..."